examples/jaeger-skaffold-trace/jaeger-all-in-one-template.yaml

```diff
@@ -0,0 +1,163 @@
+# Copyright 2021 The Skaffold Authors
```

Comment:
Since you're copying this from elsewhere, we need to keep the original copyright intact.  Any changes we bring in are copyright by us.
```suggestion
# Copyright 2017-2019 The Jaeger Authors
# Copyright 2021 The Skaffold Authors
```

Reply:
Done

Reply:
> I think this is really cool.
> 
> Since this example isn't really standalone, what do you think about turning it into a tutorial instead? And rather than maintain the jaeger-all-in-one-yaml, why not use their Helm chart? https://github.com/jaegertracing/helm-charts
> 
> Remember to copy examples/ back into integration/examples/

The linked helm chart is for a production deployment of jaeger which might be overkill for what I was intending for usage.  We can discuss what makes more sense, I am keeping this as is for now.  There is a helm chart for the jaeger-all-in-one as well maintained by a solo contributor:
https://github.com/hansehe/jaeger-all-in-one

Reply:
Done

Reply:
Keeping the same setup for now then (example dir w/ README, k8s-yaml & skaffold yaml file)